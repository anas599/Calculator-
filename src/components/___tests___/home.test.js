import { render, screen } from '@testing-library/react';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import Home from '../home';

describe('snapshot testing', () => {
  it('first snapshot', () => {
    const itemRender = renderer.create(<Home />).toJSON();
    expect(itemRender).toMatchSnapshot();
  });
});

describe('Home nav item', () => {
  it('testing home', () => {
    render(<Home />);
    const ele = screen.getByTestId('homeTest');
    expect(ele).toBeInTheDocument();
    expect(ele).toHaveTextContent('Welcome to our page');
  });
});
