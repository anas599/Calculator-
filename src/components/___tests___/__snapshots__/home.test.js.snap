// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing first snapshot 1`] = `
<div
  data-testid="homeTest"
>
  <h1>
    Welcome to our page
  </h1>
  <p
    style={
      Object {
        "textAlign": "left",
        "width": "90vw",
      }
    }
  >
    Lorem ipsum odor amet, consectetuer adipiscing elit. Lectus phasellus velit iaculis dolor porta in vestibulum risus leo. Nec tempus a tellus, nec ligula dis nibh et. Fames consectetur nibh sociosqu ante est feugiat. Nascetur nibh vestibulum suscipit porta nascetur. Nisi venenatis blandit aptent; duis efficitur facilisi. Suspendisse luctus sociosqu ac, justo per arcu. Donec etiam fermentum nunc ex cubilia fermentum pulvinar id proin?
  </p>
  <p
    style={
      Object {
        "textAlign": "left",
        "width": "90vw",
      }
    }
  >
    Lorem ipsum odor amet, consectetuer adipiscing elit. Lectus phasellus velit iaculis dolor porta in vestibulum risus leo. Nec tempus a tellus, nec ligula dis nibh et. Fames consectetur nibh sociosqu ante est feugiat. Nascetur nibh vestibulum suscipit porta nascetur. Nisi venenatis blandit aptent; duis efficitur facilisi. Suspendisse luctus sociosqu ac, justo per arcu. Donec etiam fermentum nunc ex cubilia fermentum pulvinar id proin?
  </p>
</div>
`;
